author.tf

```diff
@@ -0,0 +1,470 @@
+  docker_registry        = "${var.schema_validator_registry}"
+  container_name         = "eq-schema-validator"
+  container_port         = 5000
+  container_tag          = "${var.schema_validator_tag}"
+  healthcheck_path       = "/status"
+  application_min_tasks  = "${var.schema_validator_min_tasks}"
+  slack_alert_sns_arn    = "${module.eq-alerting.slack_alert_sns_arn}"
+}
+
+module "author-database" {
+  source                           = "./survey-runner-database"
+  env                              = "${var.env}"
+  aws_account_id                   = "${var.aws_account_id}"
+  aws_assume_role_arn              = "${var.aws_assume_role_arn}"
+  vpc_id                           = "${module.author-vpc.vpc_id}"
+  application_cidrs                = "${var.ecs_application_cidrs}"
+  multi_az                         = "${var.multi_az}"
+  backup_retention_period          = "${var.backup_retention_period}"
+  database_apply_immediately       = "${var.database_apply_immediately}"
+  database_instance_class          = "${var.database_instance_class}"
+  database_allocated_storage       = "${var.database_allocated_storage}"
+  database_free_memory_alert_level = "${var.database_free_memory_alert_level}"
+  database_name                    = "${var.author_database_name}"
+  database_user                    = "${var.author_database_user}"
+  database_password                = "${var.author_database_password}"
+  db_subnet_group_name             = "${module.author-vpc.database_subnet_group_name}"
+  database_identifier              = "${var.env}-authorrds"
+  rds_security_group_name          = "${var.env}-author-rds-access"
+}
+
+module "author-routing" {
+  source              = "./survey-runner-routing"
+  env                 = "${var.env}"
+  aws_account_id      = "${var.aws_account_id}"
+  aws_assume_role_arn = "${var.aws_assume_role_arn}"
+  public_cidrs        = "${var.public_cidrs}"
+  vpc_id              = "${module.author-vpc.vpc_id}"
+  internet_gateway_id = "${module.author-vpc.internet_gateway_id}"
+  database_subnet_ids = "${module.author-vpc.database_subnet_ids}"
+}
+
+module "author-vpc" {
+  source              = "./survey-runner-vpc"
+  env                 = "${var.env}"
+  aws_account_id      = "${var.aws_account_id}"
+  aws_assume_role_arn = "${var.aws_assume_role_arn}"
+  vpc_name            = "author"
+  vpc_cidr_block      = "${var.vpc_cidr_block}"
+  database_cidrs      = "${var.database_cidrs}"
+}
+
+module "author-survey-runner-dynamodb" {
+  source                                 = "github.com/ONSdigital/eq-terraform-dynamodb?ref=v2.0"
+  env                                    = "${var.env}-author"
+  aws_account_id                         = "${var.aws_account_id}"
+  aws_assume_role_arn                    = "${var.aws_assume_role_arn}"
+  slack_alert_sns_arn                    = "${module.eq-alerting.slack_alert_sns_arn}"
+  submitted_responses_min_read_capacity  = 1
+  submitted_responses_max_read_capacity  = 100
```

Comment:
you can remove all the these `max` restrictions. i cant see the benefit, we should also do this on the runner one.
some of the runner ones have been bumped up a little, you should also copy this